.github/workflows/remove_preview.yml

```diff
@@ -0,0 +1,22 @@
+name: Remove docs preview on PR merge
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  remove:
+    if: github.event.pull_request.merged == true
```

Comment:
Does this mean it will not be removed if it is closed without merging? (i.e. rejected)

Reply:
Yep. Now that I think about it, I suppose we want the preview to be removed as long as the PR is closed? 🤔